bip-0001.mediawiki

```diff
@@ -25,6 +25,8 @@ There are three kinds of BIP:
 
 The BIP editors assign BIP numbers and change their status. Please send all BIP-related email to the BIP editor, which is listed under [[#BIP_Editors|BIP Editors]] below. Also see [[#BIP_Editor_Responsibilities__Workflow|BIP Editor Responsibilities & Workflow]].
 
+Authors MUST NOT self assign numbers, but may use aliases such as "bip-exampleproposal".
```

Comment:
Or not? I thought there was a notice in the readme but I guess I just imagined it.

In that case I believe the notice should go in the readme after the numbers in bold. That's where people look to see that's free so you [catch them just in time](http://blog.codinghorror.com/the-just-in-time-theory/).